app/models/package_manager/maven.rb

```diff
@@ -18,15 +19,41 @@ class Maven < Base
     }.freeze
 
     def self.package_link(project, version = nil)
+      if version
```

Comment:
When would we be hitting this without a version? And are we gonna drop the ball on source then?

Reply:
I think the only spot this is actually getting called is from https://github.com/librariesio/libraries.io/blob/005d6de3e86c2354f78183dccc690a004d60fcf9/app/models/project.rb#L205

The other spot is from Base `package_link` but that has been overridden. I could remove the `if` check here and I think it would be the same result.